com.ibm.wala.cast.java.test.data/src/javaonepointfive/MoreOverriddenGenerics.java

```diff
@@ -38,15 +38,15 @@
 package javaonepointfive;
 
 public class MoreOverriddenGenerics {
-	class Super<T> {
+	static class Super<T> {
```

Comment:
Do we really want to be applying these checks and fixes to input test data?

Reply:
I mistakenly thought that this code was test infrastructure, not test input. WALA is Java code for analyzing Java code, so sometimes one gets disoriented. :blush:

I will revert this and suppress the IDE's suggestion to change it.